keps/2724-topology-aware-scheduling/README.md

```diff
@@ -977,6 +1040,45 @@ Explanation:
 
 It is worth noting that the tight fit mentioned above does not guarantee that no free capacity will be left within the assigned domains.
 
+### Cross-PodSet scheduling
+
+In consideration of a [Story 6](#story-6) a cross-podset scheduling is required,
+due to the fact that leader and workers are separate PodSets. To be able to co-locate
+leaders with its workers, two mechanisms have to be introduced:
```

Comment:
nit, maybe "we divide the problem into subproblems:"

as the bullets below are more like subproblems thatn mechanism directly.

Reply:
Good suggestion. I have rephrased the sentence.

Reply:
sgtm